cpp/ql/src/semmle/code/cpp/security/Encryption.qll

```diff
@@ -1,38 +1,50 @@
-// Common predicates relating to encryption in C and C++
+/**
+ * Provides predicates relating to encryption in C and C++.
+ */
+
 import cpp
 
-/** A blacklist of algorithms that are known to be insecure */
-string algorithmBlacklist() {
+/**
+ * Returns an algorithm that is known to be insecure.
```

Comment:
```suggestion
 * Gets the name of an algorithm that is known to be insecure.
```

https://github.com/github/codeql/blob/master/docs/qldoc-style-guide.md#predicates-with-result

Reply:
Whoops, I must've got confused here.  Thanks for the corrections!